src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,10 @@
+UART Device Emulation Module (DEM_UART)
+=======================================
+
+TODO: Add diagram showing internal structure?
+
+The UART Device Emulation Module (DEM_UART) connects to the bus of a given CPU on one side,
+and to the Debug Interconnect Interface on the other.
```

Comment:
you cannot connect to an interface, you can only connect to the debug interconnect.